active_directory/README.md

```diff
@@ -34,16 +34,18 @@ _Available for Agent versions >6.0_
 
    ```yaml
    logs:
-     - type: file
-       path: /path/to/my/directory/file.log
-       source: ruby
-       service: "<MY_SERVICE>"
```

Comment:
Should we extend the logs with the new types instead of removing the other example?

Reply:
We should not be relying on Ruby since it may send us logs in a different format breaking our pipeline and dashboards. Our native support for event viewer logs is much better suited for that purpose.